packages/url_launcher/android/src/main/java/io/flutter/plugins/urllauncher/UrlLauncherPlugin.java

```diff
@@ -79,6 +94,14 @@ private void canLaunch(String url, Result result) {
     result.success(canLaunch);
   }
 
+  private void closeWebView(Result result) {
```

Comment:
Please inline this method. We don't typically pass Result objects around unless absolutely necessary.